examples/multimodal/components/processor.py

```
@@ -204,11 +204,15 @@ async def generate(self, raw_request: MultiModalRequest):
-            user_text = raw_request.messages[0].content[0].text
-        except (IndexError, AttributeError) as e:
-            raise ValueError(f"Invalid message structure: {e}")
+        # Safely extract user text - find the text content item
+        user_text = None
+        for message in raw_request.messages:
+            for item in message.content:
+                if item.type == "text":
+                    user_text = item.text
+                    break
+        if user_text is None:
+            raise ValueError("No text content found in the request messages")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)